autogen/main/variables.tf.tmpl

```diff
@@ -488,3 +488,22 @@ variable "enable_shielded_nodes" {
   default     = true
 }
 {% endif %}
+
+
+variable "firewall_enabled" {
```

Comment:
Let's be more explicit about what firewall rules we're allowing. Maybe `add_cluster_firewall_rules`?

Reply:
Changed to your suggestion